### PR TITLE
[MLU]add mlu kernel for clip

### PR DESCRIPTION
**Pull request by Ashelly1**
### PR types

New features 

### PR changes

OPs

### Describe

add mlu kernel for clip

### CODE REVIEW

**Comment by Ashelly1** [comment]
clip op:
image

clip_grad op:
image

File: python/paddle/fluid/tests/unittests/mlu/test_clip_op_mlu.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+class TestCase5(TestClipOp):
+
+    def initTestCase(self):
+        self.dtype = np.float32
+        self.shape = (4, 8, 16)
+        self.max = 0.5
+        self.min = 0.5
+
+
```
Comment (ShawnNew) [inline python/paddle/fluid/tests/unittests/mlu/test_clip_op_mlu.py:123]: 需要覆盖一下已经注册的half和int32的kernel测试

**Review by ShawnNew — CHANGES_REQUESTED** [review]

File: python/paddle/fluid/tests/unittests/mlu/test_clip_op_mlu.py (hunk 1 above)
Comment (Ashelly1) [reply python/paddle/fluid/tests/unittests/mlu/test_clip_op_mlu.py:123]: 已增加half的kernel测试

**Review by ShawnNew — APPROVED** [review]
👍

**Comment by Ivorfeng** [comment]
👍

**Review by qili93 — APPROVED** [review]
LGTM

**Pull request merged**